Qapple/Qapple/SourceCode/Feature/0.Profile/0-1. Profile/ProfileView.swift

```diff
@@ -0,0 +1,223 @@
+//
+//  ProfileView.swift
+//  Qapple
+//
+//  Created by Simmons on 1/30/25.
+//
+
+import SwiftUI
+import MessageUI
+import ComposableArchitecture
+
+struct ProfileView: View {
+    
+    let store: StoreOf<ProfileFeature>
+    
+    var body: some View {
+        ZStack {
+            Color(Background.first)
+                .ignoresSafeArea()
+            
+            VStack(alignment: .leading, spacing: 0) {
+                NavigationBar(
+                    backgroundColor: Background.second,
+                    centerView: {
+                        Text("프로필")
+                            .font(.pretendard(.semiBold, size: 17))
+                            .foregroundStyle(.wh)
+                            .frame(width: UIScreen.main.bounds.width)
+                    }
+                )
```

Comment:
요 컴포넌트는 요렇게 변경 가능할 것 같습니다!
```suggestion
                NavigationBar(
                    title: "프로필",
                    backgroundColor: Background.second,
                )
```

Reply:
저렇게 코드를 짜면 (프로필 화면만) 뒤에 배경색이 달라서.. 정렬도 .leading으로 되어있고.. 좀 이상하게 뜨더라고요..!
<img width="200" alt="" src="https://github.com/user-attachments/assets/c658184a-8392-4fa9-a079-b6c18eeda2db" />
제 선택은 frame을 사용하는 방법이었습니다!

Reply:
헐 그러네요 컴포넌트 수정이 필요할 것 같네요,,,